defaults/main.yml

```diff
@@ -28,5 +40,26 @@ discourse_templates:
   - sshd
   - web.ratelimited
 
+# add plugins here.
+# only the url is used in the template app.yml.j2.
+# The other fields are to help you remember what they are and why you
+# what them.
+discourse_plugins:
```

Comment:
It doesn't make sense to have a long list of default plugins, every discourse is special :snowflake: but perhaps these are good examples of defaults. I'm ok with the list.